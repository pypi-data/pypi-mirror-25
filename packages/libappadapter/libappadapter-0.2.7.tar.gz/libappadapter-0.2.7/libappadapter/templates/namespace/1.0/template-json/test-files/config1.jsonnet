# Copyright 2016 Transwarp Inc. All rights reserved.

{
  name: "test",
  annotations: {
    application_params: '[{"instance_name": "dir1", "application_type": "HDFS", "store_size":
      "500G"}]',
    user_config: '{"admin": "", "normal_group": "", "admin_group": ""}',
  },

}
