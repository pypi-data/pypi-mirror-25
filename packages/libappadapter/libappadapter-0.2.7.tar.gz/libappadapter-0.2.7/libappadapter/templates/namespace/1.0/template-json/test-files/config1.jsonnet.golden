{
   "namespace.json": {
      "apiVersion": "v1",
      "kind": "Namespace",
      "metadata": {
         "annotations": {
            "application_params": "[{\"instance_name\": \"dir1\", \"application_type\": \"HDFS\", \"store_size\":\n      \"500G\"}]",
            "user_config": "{\"admin\": \"\", \"normal_group\": \"\", \"admin_group\": \"\"}"
         },
         "name": "test"
      }
   }
}
